modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -10,21 +10,29 @@ from Crypto.Hash import SHA256, SHA512
 from base64 import b64encode
 from email.utils import formatdate
 
+from {{packageName}}.configuration import Configuration
+
 HEADER_REQUEST_TARGET = '(request-target)'
 HEADER_CREATED = '(created)'
 HEADER_EXPIRES = '(expires)'
 HEADER_HOST = 'host'
 HEADER_DATE = 'date'
 HEADER_DIGEST = 'Digest'
+HEADER_AUTHORIZATION = 'Authorization'
 
 SCHEME_HS2019 = 'hs2019'
+SCHEME_RSA_SHA256 = 'rsa-sha256'
+SCHEME_RSA_SHA512 = 'rsa-sha512'
 
 ALGORITHM_RSASSA_PSS = 'PSS'
 ALGORITHM_RSASSA_PKCS1v15 = 'PKCS1v15'
 
 ALGORITHM_ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
 ALGORITHM_ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
-ALGORITHM_ECDSA_KEY_SIGNING_ALGORITHMS = {ECDSA_MODE_FIPS_186_3, ECDSA_MODE_DETERMINISTIC_RFC6979}
+ALGORITHM_ECDSA_KEY_SIGNING_ALGORITHMS = {
+    ECDSA_MODE_FIPS_186_3,
+    ECDSA_MODE_DETERMINISTIC_RFC6979
+}
 
 def get_http_signature_headers(configuration, resource_path, method, headers, body, query_params):
```

Comment:
Can you change this so you pass in signing_info rather than configuration?
The functions only use signing_info, not any other of the configuration properties.

Reply:
done. It's also using 'host', but I've simplified a bit.